packages/thirdweb/src/extensions/erc1155/write/sigMint.ts

```diff
@@ -135,7 +135,14 @@ export async function generateMintSignature(
       return "";
     })(),
     // uid computation
-    mintRequest.uid || (await randomBytesHex()),
+    ((): Hex => {
+      if (mintRequest.uid) {
+        return isHex(mintRequest.uid)
```

Comment:
A uid that is hex but not bytes32 will fail right?

Reply:
Not here but eventually yes. We could try to cast it to bytes then back to hex but that feels like too much